lib/racer.coffee

```diff
@@ -17,18 +20,61 @@ module.exports =
       type: 'string'
       default: '.source.go .comment'
       order: 3
+    show:
+      title: 'Show position for editor with definition'
+      description: 'Choose one: Right, or New'
+      type: 'string'
+      default: 'New'
+      enum: ['Right', 'New']
+      order: 4
   # members
   racerProvider: null
+  subscriptions: null
 
   activate: (state) ->
+    @subscriptions = new CompositeDisposable
 
-  provideAutocompletion: ->
+    # Register command that does find-definition
+    @subscriptions.add atom.commands.add 'atom-workspace', 'racer:find-definition': =>
```

Comment:
Am I missing something or that subscription is never released? Does Atom do it automatically?